simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountDetailFragment.kt

```diff
@@ -96,6 +108,23 @@ class AccountDetailFragment : Fragment(R.layout.account) {
   private val imageLoader: ImageLoaderType =
     services.requireService(ImageLoaderType::class.java)
 
+  private val locationPermissions = arrayOf(
+    Manifest.permission.ACCESS_FINE_LOCATION,
+    Manifest.permission.ACCESS_COARSE_LOCATION,
+  )
+
+  private val locationPermissionCallback =
+    registerForActivityResult(ActivityResultContracts.RequestMultiplePermissions()) { results ->
+      if (results.values.all { it }) {
+        openCardCreatorWebView()
+      } else {
+        showSettingsDialog()
```

Comment:
I think this is where the flow digram said that the settings dialog would only be shown if a call to shouldShowRationale() returns false, right? Otherwise this dialog will be shown before it needs to be. I would only expect it to be shown after the second time the user has denied permissions.

Reply:
Yes, you're right!